worker/include/RTC/RateCalculator.hpp

```diff
@@ -46,7 +46,7 @@ namespace RTC
 			this->newestTime      = 0u;
 			this->newestTimeIndex = -1;
 			this->oldestTimeIndex = 0u;
-			this->oldestIndex     = -1;
+			this->oldestTimeIndex = -1;
```

Comment:
This and previous line set the same member.

Reply:
Fixed.